corehq/apps/case_search/templates/case_search/bootstrap3/profile_case_search.html

```diff
@@ -1,82 +0,0 @@
-{% extends 'hqwebapp/bootstrap3/base_section.html' %}
-{% load hq_shared_tags %}
-{% load compress %}
-{% load i18n %}
-
-{% requirejs_main 'case_search/js/bootstrap3/profile_case_search' %}
-
-{% block page_content %}
-
-<div id="profile-case-search">
-  <h3>Profile Slow Case Searches</h3>
-  <p>
-    Paste in a JSON representation of the request dict made to case search, such
-    as that found in 'LongCaseSearchRequest' error logs
-  </p>
-  <div>
-    <form>
-      <div class="form-group">
-        <label for="appId">App ID (if not already in the request dict)</label>
-        <input data-bind="value: appId" type="text" class="form-control">
-      </div>
-      <div class="form-group">
-        <label for="requestDict">request_dict</label>
-        <textarea id="requestDict" spellcheck="false" class="form-control vertical-resize" name="request_dict"
-          type="json" data-bind="value: requestDict, attr: {placeholder: exampleRequestDict}"></textarea>
-      </div>
-      <button class="btn btn-primary" data-bind="click: search">
-        <i data-bind="attr: {class: searchButtonIcon}"></i> {% trans "Search" %}
-      </button>
-    </form>
-  </div>
-
-  <div class="spacer"></div>
-
-  <div data-bind="if: results">
-    <p>Number of primary results: <span data-bind="text: results().primary_count"></span></p>
-    <p>Number of related cases: <span data-bind="text: results().related_count"></span></p>
-    <p>Total browser runtime: <span data-bind="text: browserTime"></span> seconds</p>
-    <p>Total server runtime: <span data-bind="text: results().timing_data.duration.toFixed(3)"></span> seconds</p>
-
-    <ul data-bind="template: {name: 'timingRow', data: results().timing_data }"
-      class="list-group"></ul>
-
-    <script id="timingRow" type="text/html">
-      <li class="list-group-item">
-        <div class="container-fluid">
-          <div class="row">
-            <div class="col-xs-3" data-bind="text: name"></div>
-            <div class="col-xs-3">
-              <strong data-bind="text: duration ? duration.toFixed(3) : '-'"></strong> seconds;
-            </div>
-            <div class="col-xs-3">
-              <strong data-bind="text: percent_total ? percent_total.toFixed(3) : '-'"></strong>% of total;
-            </div>
-            <div class="col-xs-3">
-              <strong data-bind="text: percent_parent ? percent_parent.toFixed(3) : '-'"></strong>% of parent
-            </div>
-          </div>
-          <ul data-bind="template: { name: 'timingRow', foreach: subs }"
-          class="list-group"></ul>
-        </div>
-      </li>
-    </script>
-
-    <div class="panel-group" id="queries-accordion" data-bind="foreach: results().queries">
```

Comment:
Ah, I see the queries section is present here, but not in the `profile_case_search.html` that remains

Reply:
Hi Ethan, good finding. This panel is removed in split files stage... I'm still looking into why it happened. 😭 This is the commit db1b8b64b226f after I first merge master into my branch, and from the diff we can see your change is marked as diff...